SMS traffic pumping fraud FAQ (#7475)
braze-inc/braze-docs: _docs/_hidden/private_betas/sms_traffic_pumping_fraud.md

---
nav_title: SMS Traffic Pumping Fraud FAQs
permalink: "/sms_traffic_pumping_fraud/"
description: "This reference article covers frequently asked questions for SMS traffic pumping fraud."
hidden: true
---

# SMS traffic pumping fraud FAQs 

### What is SMS traffic pumping fraud? 

> SMS traffic pumping is a rising threat in the SMS space. It occurs when fraudsters find a way to trigger SMS message sends to phone numbers that are not associated with real customers in order to collect revenue tied to fraudulent message sending. Most often, they trigger high volume SMS sends using online forms, e.g. for SMS opt-ins or one-time passwords for password resets or account login.  

Reviewer: ```suggestion
SMS traffic pumping is a rising threat in the SMS space. It occurs when fraudsters find a way to trigger SMS message sends to phone numbers that are not associated with real customers in order to collect revenue tied to fraudulent message sending. Most often, they trigger high-volume SMS sends using online forms, such as forms for SMS opt-ins or one-time passwords for password resets or account login.  
```

### What does Braze do to mitigate SMS pumping fraud?

Braze currently maintains an SMS blocklist for both US-embargoed countries, as well as for countries known to be high risk for traffic pumping, which can be referenced in [our documentation]({{site.baseurl}}/sms_country_blocklist). All attempted sends to phone numbers with these country codes are blocked.

Additionally, Braze is introducing an SMS Geographic Permission Allowlist, which will further protect against fraudulent behavior by enforcing controls on what countries you are able to send to.

### How can I protect my brand against SMS pumping fraud? 

There are several ways you can protect yourself, including: 
- **Monitor your daily SMS sending volumes for spikes and abnormalities:**

>     - We recommend customers set [campaign limits and alerts]({{site.baseurl}}/user_guide/engagement_tools/campaigns/managing_campaigns/campaign_alerts/) to cap and notify if an unusually high number of messages are sent.

Reviewer: ```suggestion
    - We recommend you set [campaign limits and alerts]({{site.baseurl}}/user_guide/engagement_tools/campaigns/managing_campaigns/campaign_alerts/) to cap and notify if an unusually high number of messages are sent.
```

    - Unusual spikes in sending of messages might indicate traffic pumping.
    - Unusually high number of opt-ins in a short time frame (outside of intentional strategies to drive opt-ins) might indicate traffic pumping.
- **Improve security for online phone number capture forms:**
    - Braze [SMS sign-up form templates]({{site.baseurl}}/user_guide/message_building_by_channel/in-app_messages/drag_and_drop/templates/phone_number_capture) provide out-of-the box security measures, such as validating the phone number length and format. You can also set up the form to only collect phone numbers with country codes that align to your target customers:
        - For example, if you only do business in the US and UK, set up the form to only collect numbers with a +1 and +44 country code (technical details can be found in [our documentation]({{site.baseurl}}/user_guide/message_building_by_channel/in-app_messages/drag_and_drop/templates/phone_number_capture/#step-2-customize-your-phone-number-input-component)).

>     - If you are building a custom phone number capture on your website, we recommend setting rules to validate phone number length and format and ensuring forms are fully complete before collecting phone numbers. Be sure to work with your engineering or technical team to validate the form inputs both client-side and server-side for maximum protection.

Reviewer: ```suggestion
    - If you are building a custom phone number capture on your website, we recommend setting rules to validate phone number length and format and confirming that forms are fully complete before collecting phone numbers. Be sure to work with your engineering or technical team to validate the form inputs both client-side and server-side for maximum protection.
```
